ui/app/src/App.tsx

```diff
@@ -51,12 +55,12 @@ function App() {
                 "0x1C277bD41A276F87D3E92bccD50c7364aa2FFc69",
                 3,
                 usePublicClient(),
-                useWalletClient()?.data as WalletClient,
+                useWalletClient()?.data as WalletClient
               )
             }
           />
         </li>
-      </ul>
+      </ul> */}
```

Comment:
If this is giving you troubles, can probably remove and add in a separate PR

NOTE: the `StreamStatus` component is supposed to help display the amount of stream time left for something like the deployment list page in chaosnet (as one field in the display of each item)

Reply:
ok